operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_CPS.md

```diff
@@ -101,6 +94,23 @@ BOS will construct it in the following format:
 root=craycps-s3:s3-path:<etag>:<transport>:<api_gateway>:<timeout>:interface[,<interface>[,<interface>]...]:<ramroot>
 ```
 
+### Example kernel parameter
+
+```text
+root=craycps-s3:s3://boot-images/079f6bce-d902-438a-8db7-bcb06a21d3de/rootfs:727f630b5a0953c21026eeca6db7e49f-1792:dvs:api-gw-service-nmn.local:300:hsn0,nmn0:0
+```
+
+## Boot set S3 parameters
+
+The session template boot set contains several [Simple Storage Service (S3)](../../glossary.md#simple-storage-service-s3) parameters.
+These are listed below, along with the appropriate values to use.
+
+- `type`: Set to `s3`
+- `path`: Set to `s3://<BUCKET_NAME>/<KEY_NAME>/manifest.json`
+  - `<BUCKET_NAME>` is set to `boot-images`
+  - `<KEY_NAME>` is set to the image ID that the [Image Management Service (IMS)](../../glossary.md#image-management-service-ims) created when it generated the boot artifacts.
+- `etag`: is set to `<etag\>` of the manifest.json file that [Image Management Service (IMS)](../../glossary.md#image-management-service-ims) created when it generated the boot artifacts.
```

Comment:
Why is the ETag written as `<etag\>` here? Why the angle brackets and backslash? Perhaps it could be rephrased as:

> The ETag of the `manifest.json` file in S3 as stored by the [Image Management Service (IMS)](../../glossary.md#image-management-service-ims).

Reply:
No idea. Changed.